.changeset/tidy-feet-wash.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/badge': patch
+'@twilio-paste/core': patch
+---
+
+[Badge] Fixed the type of the children prop
```

Comment:
❓ Should this be a patch or a minor version? I made it a patch for now because the type is more inclusive now.

Reply:
Patch is appropriate. You're fixing a thing. If you were adding noteworthy new features, it'd be a minor